src/lens/get-timeline.js

```diff
@@ -312,6 +312,7 @@ const getTimelineRequest = (request) => {
   return apolloClient.query({
     query: gql(GET_TIMELINE),
     variables: { request },
+    fetchPolicy: "network-only",
```

Comment:
We can't just put 'network-only' on every apollo query. It means we would never use apollo cache. For example, every time you click on Explore (feed) then do something and go back, the feed will take as much time to load as the first time. this is not a long term solution, sorry